app/templates/carpools/show.html

```diff
@@ -2,78 +2,153 @@
 {%- import "bootstrap/wtf.html" as wtf %}
 
 {% block site %}
-<div class='page-header'>
-    <h1>Carpool Details</h1>
-</div>
+<div class="content">
+
+    <div class="fullscreen">
+        <form class="form-page">
+
+            {% set current_user_ride_request = pool.get_current_user_ride_request() %}
+
+            <h4>Carpool Details</h4>
+            <h1>{{ pool.from_place }} to {{ pool.destination.name }}</h1>
+
+            <h4>Carpool Status</h4>
+
+        {% if current_user_ride_request and current_user_ride_request.status == 'approved'  %}
+            <div class="two-col-layout">
+                <div class="two-col-column">
+                    <p>You are confirmed for this carpool!</p>
+                </div>
+                <div class="two-col-column">
+                    <a href="{{ url_for('carpool.modify_ride_request', carpool_uuid=pool.uuid, request_uuid=current_user_ride_request.uuid, action='cancel') }}"><input class="destructive" type="submit" value="Cancel your seat in carpool"></a>
```

Comment:
These `<input>`s inside of `<a href>`s won't work. They need to be styled `<a href>`.

Reply:
(They won't work because the browser will just go back to the carpool details page without doing anything.)